### PR TITLE
Adding support for exogenously defined waste incineration CCS shares and disabling feedstock emissions with unknown fate accounting

### PR DESCRIPTION
**Pull request by Renato-Rodrigues**
## Purpose of this PR

1. Add support for exogenously defined waste incineration CCS share:
```
*** cm_wasteIncinerationCCSshare, proportion of waste incineration that is captured at a given year and region
*** off: means that all plastics incineration emissions in the World goes back to the atmosphere.
*** 2050.GLO 0.5, 2050.EUR 0.8: means that 50% of waste incineration emissions are captured for all regions from 2050 onward, except for Europe that has 80% of its waste incineration emissions captured.
*** The capture rate increases linearly from zero, in 2025, to the value set at the switch.
$setglobal cm_wasteIncinerationCCSshare  off      !! def = off
```

2. Add support for disabling chemicals feedstock emissions with unknown fate accounting
```
*** cm_feedstockEmiUnknownFate, account for chemical feedstock emissions with unknown fate
*** off: assume that these emissions are trapped and do not account for total anthropogenic emissions 
*** on: account for chemical feedstock emissions with unknown fate as re-emitted to the atmosphere
$setglobal cm_feedstockEmiUnknownFate  off      !! def = off
```

## Type of change

- [x] New feature 

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)

## Todo: 
@mellamoSimon 

- [x] Define the incineration CCS share values that needs to be used for the default mitigation scenarios and add them to the scenario config for the validation.
e.g.:

| title | cm_wasteIncinerationCCSshare |
| -- | -- |
| SSP2EU-NPi-calibrate | |  
| SSP2EU-Base | |
| SSP2EU-NDC |  |
| SSP2EU-NPi |  |
| SSP2EU-PkBudg500 |  ~~2050.GLO 0.8~~ 2050.GLO 0.9 |
| SSP2EU-PkBudg650 |  ~~2050.GLO 0.65~~ 2050.GLO 0.9  |
| SSP2EU-PkBudg1050 |  2050.GLO 0.5 |
...

- [ ] do test runs for the new validation including aceptable values for the previously mentioned CCS shares
- [ ] do two test runs to check if the `cm_feedstockEmiUnknownFate` feature works for both situations, with it set to `off` and set to `on`

### CODE REVIEW

**Comment by mellamoSimon** [comment]
Thank you Renato for the development. I suggest that we use the values below. My reasoning is: that reported costs for CCS at the CO2 concentration level of incineration plants should be between 100-200 USD/tCO2, well below typical carbon prices for PkBudg650 policy runs. So we can go with the technical maximum capture rate of 90%. PkBudg1050 seems to be in the same ballpark that CCS costs in the latest successful validation runs, so I think .5 is a good guess to start with (basically we cannot say much here). My sources could be better (rn I'm using figures compiled by the "global CCS institute") but we can tweak the values after we get new runs.


title | cm_wasteIncinerationCCSshare
-- | --
SSP2EU-NPi-calibrate |  2050.GLO 0
SSP2EU-Base |  2050.GLO 0
SSP2EU-NDC |  2050.GLO 0
SSP2EU-NPi |  2050.GLO 0
SSP2EU-PkBudg500 | 2050.GLO 0.9
SSP2EU-PkBudg650 | 2050.GLO 0.9
SSP2EU-PkBudg1050 | 2050.GLO 0.5

**Comment by LaviniaBaumstark** [comment]
thank you for implementing this so quick!

**Comment by LaviniaBaumstark** [comment]
@orichters @RahelMA could you please check, if this switch needs to be added to scenario_config_ELEVATE4p4.csv ? Thanks

**Comment by orichters** [comment]
I would suggest in the ELEVATE config to add the specification used for NPi to the two `ELV_CurPol*` runs, something used for NDC to all `ELV_NDC2030*` runs and something that suits a 1.5°C scenario to `ELV_NPi2020_700*` and something corresponding to 2°C for the `ELV_LTS*` runs.

**Comment by orichters** [comment]
Thanks!

**Review by LaviniaBaumstark — COMMENTED** [review]
did you run `make test`?

**Review by fschreyer — APPROVED** [review]

File: main.gms [hunk 1]
```diff
@@ -1543,7 +1543,15 @@ $setGlobal cm_CESMkup_build  standard  !! def = standard
 *** addressed in cm_CESMkup_ind_data.
 $setGlobal cm_CESMkup_ind        standard  !! def = standard
 $setGlobal cm_CESMkup_ind_data   ""        !! def = ""
-
+*** cm_wasteIncinerationCCSshare, proportion of waste incineration that is captured at a given year and region
+*** off: means that all plastics incineration emissions in the World goes back to the atmosphere.
+*** 2050.GLO 0.5, 2050.EUR 0.8: means that 50% of waste incineration emissions are captured for all regions from 2050 onward, except for Europe that has 80% of its waste incineration emissions captured.
+*** The capture rate increases linearly from zero, in 2025, to the value set at the switch.
```
[the remark this answers is not in the document]
Comment (Renato-Rodrigues) [reply main.gms]: https://github.com/remindmodel/remind/pull/1631/commits/ba2b8c6f3b2d0ebc60cb156d1503f06b6a267f81

File: modules/37_industry/subsectors/equations.gms [hunk 2]
```diff
@@ -296,8 +296,10 @@ q37_incinerationEmi(t,regi,sefe(entySe,entyFe),emiMkt)$(
                          entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt)) ..
   vm_incinerationEmi(t,regi,entySe,entyFe,emiMkt)
   =e=
+  (
     v37_plasticWaste(t,regi,entySe,entyFe,emiMkt)
   * pm_incinerationRate(t,regi)
+  ) * (1 - p37_regionalWasteIncinerationCCSshare(t,regi))
```
[the remark this answers is not in the document]
Comment (Renato-Rodrigues) [reply modules/37_industry/subsectors/equations.gms:302]: https://github.com/remindmodel/remind/pull/1631/commits/ba2b8c6f3b2d0ebc60cb156d1503f06b6a267f81

File: modules/37_industry/subsectors/equations.gms [hunk 3]
```diff
@@ -317,8 +319,12 @@ q37_feedstockEmiUnknownFate(t,regi,sefe(entySe,entyFe),emiMkt)$(
                          entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt) ) ..
   vm_feedstockEmiUnknownFate(t,regi,entySe,entyFe,emiMkt)
   =e=
+$ifthen.cm_feedstockEmiUnknownFate not "%cm_feedstockEmiUnknownFate%" == "off"
+  (
     v37_FeedstocksCarbon(t,regi,entySe,entyFe,emiMkt)
   * (1 - s37_plasticsShare)
+  )
+$endIf.cm_feedstockEmiUnknownFate
```
[the remark this answers is not in the document]
Comment (Renato-Rodrigues) [reply modules/37_industry/subsectors/equations.gms:329]: https://github.com/remindmodel/remind/pull/1631/commits/ba2b8c6f3b2d0ebc60cb156d1503f06b6a267f81

**Review by LaviniaBaumstark — APPROVED** [review]

**Comment by Renato-Rodrigues** [comment]
> did you run `make test`?

Yes I did, although this does not guarantee everything, as one of these switches is only active in ambitious policy runs.

**Pull request merged**